frontend/src/Helpers/Utils.ts

```diff
@@ -1,3 +1,9 @@
 export function Average(arr: number[]): number {
   return arr.reduce((a, b) => a + b) / arr.length;
 }
+export function Max(arr: number[]): number {
+  return arr.reduce((a, b) => Math.max(a, b));
+}
+export function Min(arr: number[]): number {
+  return arr.reduce((a, b) => Math.min(a, b));
+}
```

Comment:
I might be wrong but you can just do
```ts
Math.max(...arr)
Math.min(...arr)
```

So I guess this utils are not needed and can just use the native `Math` directly?

Reply:
you are probably right oops, i'll get that changed